src/views/Document.vue

```diff
@@ -7,38 +7,38 @@
 			"
 			description="Nižšie nájdete podrobné štatistiky kontroly originality vašej práce"
 		/>
-		<div v-if="document" class="px-6 md:px-0 container mx-auto my-20">
+		<div v-if="document" class="px-0 mx-1 md:px-6 md:container md:mx-auto mt-4 md:my-20">
 			<a
 				v-if="isMultiple"
-				class="flex items-center p-2 mb-2 cursor-pointer text-2xl text-gray-600 hover:text-gray-700"
+				class="flex items-center px-3 py-2 md:p-2 mb-0 md:mb-2 cursor-pointer text-lg md:text-2xl text-gray-600 hover:text-gray-700"
```

Comment:
The items are super small.
![image](https://user-images.githubusercontent.com/27869979/109663855-d67af980-7b6c-11eb-86d0-7aaa4cee6498.png)